app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command, old_file, new_file = command.split()
```

Comment:
DRY 

Create separate variable for command.split() to avoid using this function twice